recipe/meta.yaml

```diff
@@ -1,18 +1,15 @@
-{% set version = "1.4.9" %}
+{% set version = "1.4.10" %}
 
 package:
   name: kealib
   version: {{ version }}
 
 source:
   url: https://bitbucket.org/chchrsc/kealib/downloads/kealib-{{ version }}.tar.gz
-  sha256: 1c80489f17114a229097c2e8c61d5e4c82ea63ae631c81a817fef95cfd527174
-  patches:
-    # by default kealib forces C++98 mode which breaks other things built in C++11 mode
-    - disablecpp98.patch
+  sha256: b1bd2d6834d2fe09ba456fce77f7a9452b406dbe302f7ef1aabe924e45e6bb5e
 
 build:
-  number: 3
+  number: 1003
```

Comment:
Since this is a new version it should be `1000`. The bot uses current + 1000, but you updated it from `1.4.9` to `1.4.10`.